libraries/DataFlash/LogFile.cpp

```diff
@@ -366,8 +366,7 @@ void DataFlash_Class::Log_Write_RSSI(AP_RSSI &rssi)
     WriteBlock(&pkt, sizeof(pkt));
 }
 
-// Write a BARO packet
-void DataFlash_Class::Log_Write_Baro(AP_Baro &baro, uint64_t time_us)
+void DataFlash_Class::Log_Write_Baro_instance(AP_Baro &baro, uint64_t time_us, uint8_t baro_instance, enum LogMessages type)
 {
     if (time_us == 0) {
         time_us = AP_HAL::micros64();
```

Comment:
Keeping this in the *instance* version of the method means that if `time_us` isn't provided to the *generic* method, then the baro messages will have different time. I don't have a strong preference, but I think that keeping all messages with the same time is better, so I suggest this piece of code should be in the *generic* method.